tests/utils.py

```diff
@@ -241,3 +244,55 @@ def teardown_spark():
 
 def _scala_test_jar_url():
     return s3.http_url(os.path.basename(os.environ["SCALA_TEST_JAR_PATH"]))
+
+
+def spark_security_session():
+    '''
+    Spark strict mode setup is slightly different from dcos-commons, so can't use sdk_security::security_session.
```

Comment:
explain why?

Reply:
I'll add more details about what's different: (1) the role is "*", (2) the driver permissions to execute tasks.